src/serve/grpc/query.rs

```diff
@@ -215,6 +218,38 @@ fn into_u5c_utxo(
     })
 }
 
+fn into_u5c_era_summary(summary: EraSummary) -> u5c::cardano::EraSummary {
+    let start = summary.start;
+    let end = summary.end.unwrap();
+    let pparams = summary.pparams;
+
+    u5c::cardano::EraSummary {
+        name: match_pparams_to_era_name(pparams),
+        start: Some(u5c::cardano::EraBoundary {
+            time: start.timestamp.timestamp() as u64,
+            slot: start.slot,
+            epoch: start.epoch,
+        }),
+        end: Some(u5c::cardano::EraBoundary {
+            time: end.timestamp.timestamp() as u64,
+            slot: end.slot,
+            epoch: end.epoch,
+        }),
+        config: "".to_string(),
```

Comment:
This should be the JSON for the era. In Dolos, you can get this from `byron_path`